handbook/engineering/deployments/postgresql.md

```diff
@@ -1,25 +1,70 @@
-# PostgreSQL database tips
+# PostgreSQL deployments
 
-### Shell commands
+For deployments other than Cloud and Sourcegraph.com please use the information [here](https://docs.sourcegraph.com/admin/faq#how-do-i-access-the-sourcegraph-database) to access the database.
 
-These commands assume you're on a local machine, and trying to access the live systems. Also refer to the [deployment page's Kubernetes section](./kubernetes.md) for kubectl tips.
+## Sourcegraph.com specific
 
-### Sourcegraph.com specific
+We currently run two seperate databases. The `sg-cloud` database is the primary database and the `sg-cloud-code-intel` is used by the code-intel team.
 
 You can also directly view database in [GCP](https://console.cloud.google.com/sql/instances?project=sourcegraph-dev)
 
-We currently run two seperate databases. The `sg-cloud` database is the primary database and the `sg-cloud-code-intel` is used by the code-intel team.
+We utilize the GCP utility of the [Cloud SQL Proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) to connect to our production databases. By default, our Cloud SQL databases are not accessible without this proxy.
+
+For read-only access there is also an option of using [BigQuery](
+https://console.cloud.google.com/bigquery?sq=527047051561:67f2616f4acb4b7cb3639e4a97e2f4aa
+) and their `EXTERNAL_QUERY` syntax.
+
+There are two ways of connecting: either using the `gcloud beta sql connect` command or running the proxy on a port locally.
```

Comment:
As a developer, how do I know which way to choose?

Reply:
gcloud is mainly for quick `pgsql` commands, the proxy is used when you want to connect an external database viewer or GUI program.

Reply:
^ I will update it with this info

Reply:
```suggestion
There are two ways of connecting: either using the `gcloud beta sql connect` command which will use the `pgsql` client or running the proxy on a port locally to utilize whatever tools you typically use.

```